azurerm/resource_arm_application_gateway.go

```diff
@@ -630,6 +641,67 @@ func resourceArmApplicationGateway() *schema.Resource {
 				},
 			},
 
+			"ssl_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"disabled_protocols": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							Elem: &schema.Schema{
+								Type:             schema.TypeString,
+								DiffSuppressFunc: suppress.CaseDifference,
+								ValidateFunc: validation.StringInSlice([]string{
+									string(network.TLSv10),
+									string(network.TLSv11),
+									string(network.TLSv12),
+								}, true),
```

Comment:
since this is a new field can we make this case sensitive by making this false?

```suggestion
								}, false),
```